src/CruxComponent.tsx

```diff
@@ -190,35 +194,47 @@ export class CruxComponentCreator {
                             <Table className="table table-striped cftable" striped bordered condensed hover>
                                 <thead>
                                 <tr>
-                                    {_.map(constants.fields.filter((field: any) => field.display), (field: any, index: any) =>
+                                    {constants.fields.filter((field: any) => field.display).map((field: any, index: any) =>
```

Comment:
throughout the codebase we prefer lodash for filter, map, find kind of operations. It helps us stay away from nullpointers (calling function filter of undefined). Lets not change this. Infact lets change the filter also to lodash